admin/js/src/shareable-canvas.js

```diff
@@ -217,30 +211,42 @@ class ShareableCanvas {
 
   }
 
-  loadImage(path) {
+  drawBackground() {
+    let background = new createjs.Shape();
+
+    background.graphics.beginFill("Black").drawRect(0,0, this.canvas.width, this.canvas.width);
+    this.stage.addChild(background);
+    this.update();
+  }
+
+  loadImage(path, fixSize) {
     let that = this;
     let image = new Image();
     image.src = path;
 
-    image.onload = function(event) {
+    image.onload = (event, fixSize) => {
       let loadedImage = event.target;
-      that.addImage(loadedImage);
+      this.addImage(loadedImage);
     }
 
   }
 
   addImage(image) {
-    let that = this;
     let bitmap = new createjs.Bitmap(image);
 
+    bitmap.alpha = 0.3;
+
     let bounds = bitmap.getBounds();
-    bitmap.regX = bounds.width / 2;
-    bitmap.regY = bounds.height / 2;
+    let scale = this.canvas.height / bounds.height;
+
+    bitmap.scaleY = scale;
+    bitmap.scaleX = scale;
 
     bitmap.addEventListener('mousedown', event => this.savePosition(event));
     bitmap.addEventListener('pressmove', event => this.transform(event));
```

Comment:
needs some scaling or something otherwise this move isn't useful as the image will be almost exactly the same size as the container. [except portrait images or similar where perhaps moving the image will be more useful so maybe it doesnt matter]